framework/src/media/audio/audio_manager.c

```diff
@@ -143,7 +143,7 @@ struct audio_card_info_s {
 	uint8_t device_id;			//current device id
 	struct audio_device_config_s config[CONFIG_AUDIO_MAX_DEVICE_NUM];
 	struct pcm *pcm;
-	audio_manager_stream_policy_t policy;
+	stream_policy_t policy;
```

Comment:
stream_info_t

Reply:
Modify audio_card_info to own stream_info_t

Reply:
This PR is for adding stream_info_t.
I'll modify it after this PR.